Documentation/Administration/Production/ReverseProxy/Index.rst

```diff
@@ -0,0 +1,159 @@
+:navigation-title: Reverse Proxy
+
+..  include:: /Includes.rst.txt
+.. _reverse-proxy-setup:
+
+====================================
+Running TYPO3 behind a reverse proxy
```

Comment:
Technically speaking there is a difference between a reverse proxy and the mentioned cloud load balancers. 

From a Typo3 point of view there is no difference. But you want to mention both in the title of the page. So something like "running Typo3 behind a reverse proxy or loadbalancer"

This page will cover both, mentioning both makes sure people can find the page.

Reply:
good idea!

Reply:
I changed the title